cargo-concordium/src/main.rs

```diff
@@ -200,6 +200,56 @@ A schema has to be provided either as part of a smart contract module or with th
         )]
         module_path:  Option<PathBuf>,
     },
+    #[structopt(
+        name = "schema-template",
+        about = "Convert a schema into its template representation and output it to a file or \
+                 print it to the console.
+A schema has to be provided either as part of a smart contract module or with the schema flag. You \
+                 need to use exactly one of the two flags(`--schema` or `--module`) with this \
+                 command."
+    )]
+    SchemaTemplate {
+        #[structopt(
+            name = "out",
+            long = "out",
+            short = "o",
+            default_value = "-",
+            help = "Path and filename to write the converted template representation to or use \
+                    the default value `-` to print the template schema to the console. The path \
+                    has to exist while the file will be created. (expected input: \
```

Comment:
Why do we not create the path? Don't we actually do that already?

Reply:
yes, we create the path already. I removed that sentence and also removed the sentence in `SchemaBase64` command.